src/ProjectTemplates/Web.Spa.ProjectTemplates/content/Angular-CSharp/Startup.cs

```diff
@@ -88,7 +88,10 @@ public void Configure(IApplicationBuilder app, IWebHostEnvironment env)
 
 #endif
             app.UseStaticFiles();
-            app.UseSpaStaticFiles();
+            if (env.IsProduction())
```

Comment:
Actually, I'm wondering whether this should be `env.IsProduction()` or `!env.IsDevelopment()`?